app/src/main/java/com/github/sdp/ratemyepfl/fragment/navigation/MapFragment.kt

```diff
@@ -1,19 +1,22 @@
 package com.github.sdp.ratemyepfl.fragment.navigation
 
+import android.content.Intent
 import android.os.Bundle
 import android.view.View
 import android.widget.Button
+import androidx.fragment.app.Fragment
 import com.github.sdp.ratemyepfl.R
 import com.github.sdp.ratemyepfl.activity.map.MapActivity
 
-class MapFragment : TabFragment(R.layout.fragment_map) {
+class MapFragment : Fragment(R.layout.fragment_map) {
```

Comment:
Are we going to integrate the map in the fragment ?

Reply:
The plan was to display the map on the fragment. I do not know what Nicolas does for this sprint, but the button will be removed later for sure. It was just wrongly marked as a TabFragment.